pkg/apis/deviceplugin/v1/webhook_common.go

```diff
@@ -51,6 +51,8 @@ type commonDevicePluginValidator struct {
 var _ admission.CustomValidator = &commonDevicePluginValidator{}
 
 // Default implements admission.CustomDefaulter so a webhook will be registered for the type.
+//
+//nolint:gocyclo
```

Comment:
no configuration for this check that'd allow longer `switch`  checks?

Reply:
I'll see. The lint error was about the cyclomatic complexity.